src/main/java/Arguments/TranscriptArguments.java

```diff
@@ -0,0 +1,53 @@
+package Arguments;
+
+import Transcription.VoskAdapter;
+
+import java.util.List;
+
+public class TranscriptArguments {
```

Comment:
These files are generated by Lombok; it would be better to just have the annotation instead of delomboking this and the other files.

Reply:
Removed arguments DTOs since each adapter (Transcript, Search, and PrintAdapter) are all built in the argument parser anyway. Also fixed any delombokked files.